TrackMania 2D/src/tm2DGame/terrain/Terrain.java

```diff
@@ -10,17 +10,17 @@
 	int x, y;
 	Image image;
 	
-	//un terrain ne bouge pas. on n'a pas besoin de red�finir un nouveau rectangle � chaque 
+	//un terrain ne bouge pas. on n'a pas besoin de red�finir un nouveau rectangle � chaque 
```

Comment:
WHAT IS ENCODAGE ? 
BABY DON'T HURT ME
DON'T HURT ME
NO MORE

Reply:
c'était pas mon commentaire à la base :p